packages/api-server/setup.py

```diff
@@ -38,5 +38,8 @@
             "rmf_api_server=api_server.__main__:main",
         ],
     },
+    package_data={
+        "static": ["*"],
+    },
```

Comment:
the files are not exported, can test by running `pnpm run prepack`. According to the docs the format should be 
```python3
package_data={"mypkg": ["*.txt", "*.rst"]}
```
https://setuptools.pypa.io/en/latest/userguide/datafiles.html

Reply:
yeah that's a good idea, I was having trouble with `setup.py` to handle those files properly, they are in the `api_server` package and behaving now